server/node/src/main/java/org/kaaproject/kaa/server/control/service/DefaultControlService.java

```diff
@@ -1015,6 +1019,56 @@ public void onNodeAdded(OperationsNodeInfo node) {
     return resolver.getNode(entityId);
   }
 
+  /**
+   * Used when writing OperationsNodeInfo instance in logs. OperationsNodeInfo instance has
+   * ByteBuffer fields which written incorrectly in the log file. In this method ByteBuffer fields
+   * temporary overwrite to null, print logs, and after ByteBuffer fields recover on theirs previous
+   * values.
+   *
+   * @param logString is a usual string for logging. Use '{}' for output OperationsNodeInfo and
+   *                  the second '{}' is OperationsServerResolver
+   * @param node      in which find and temporary overwrite all ByteBuffer-s to null
+   * @param resolver  OperationsServerResolver instance
```

Comment:
And why did you set these fields to null instead of applying hex conversion on them?